pkg/fanal/analyzer/imgconf/apk/apk_test.go

```diff
@@ -1508,86 +1509,41 @@ func TestResolveDependency(t *testing.T) {
 	var tests = map[string]struct {
 		pkgName             string
 		apkIndexArchivePath string
-		expected            map[string]struct{}
+		want                set.Set[string]
 	}{
 		"low": {
 			pkgName:             "libblkid",
 			apkIndexArchivePath: "testdata/history_v3.9.json",
-			expected: map[string]struct{}{
-				"libblkid": {},
-				"libuuid":  {},
-				"musl":     {},
-			},
+			want:                set.New("libblkid", "libuuid", "musl"),
 		},
 		"medium": {
 			pkgName:             "libgcab",
 			apkIndexArchivePath: "testdata/history_v3.9.json",
-			expected: map[string]struct{}{
-				"busybox":  {},
-				"libblkid": {},
-				"libuuid":  {},
-				"musl":     {},
-				"libmount": {},
-				"pcre":     {},
-				"glib":     {},
-				"libgcab":  {},
-				"libintl":  {},
-				"zlib":     {},
-				"libffi":   {},
-			},
+			want: set.New("busybox", "libblkid", "libuuid", "musl", "libmount", "pcre", "glib",
+				"libgcab", "libintl", "zlib", "libffi"),
 		},
 		"high": {
 			pkgName:             "postgresql",
 			apkIndexArchivePath: "testdata/history_v3.9.json",
-			expected: map[string]struct{}{
-				"busybox":               {},
-				"ncurses-terminfo-base": {},
-				"ncurses-terminfo":      {},
-				"libedit":               {},
-				"db":                    {},
-				"libsasl":               {},
-				"libldap":               {},
-				"libpq":                 {},
-				"postgresql-client":     {},
-				"tzdata":                {},
-				"libxml2":               {},
-				"postgresql":            {},
-				"musl":                  {},
-				"libcrypto1.1":          {},
-				"libssl1.1":             {},
-				"ncurses-libs":          {},
-				"zlib":                  {},
-			},
+			want: set.New("busybox", "ncurses-terminfo-base", "ncurses-terminfo", "libedit", "db", "libsasl",
+				"libldap", "libpq", "postgresql-client", "tzdata", "libxml2", "postgresql", "musl", "libcrypto1.1",
+				"libssl1.1", "ncurses-libs", "zlib"),
 		},
 		"package alias": {
 			pkgName:             "sqlite-dev",
 			apkIndexArchivePath: "testdata/history_v3.9.json",
-			expected: map[string]struct{}{
-				"sqlite-dev":  {},
-				"sqlite-libs": {},
-				"pkgconf":     {}, // pkgconfig => pkgconf
-				"musl":        {},
-			},
+			want: set.New(
```

Comment:
nit: Let's use same fmt.

Reply:
We need to keep a comment in this test case. Instead, I reverted others. https://github.com/aquasecurity/trivy/pull/8149/commits/a69fd6645ff8bc2936e024426e454a3b4f83c235